### PR TITLE
docs: Add banner message about Devstack being deprecated

### PR DESCRIPTION
**Pull request by sarina**
Addresses https://github.com/openedx/axim-engineering/issues/1173

### CODE REVIEW

**Comment by sarina** [comment]
@robrap - I don't think docs run on devstack builds, but does this banner wording work for you?

**Comment by robrap** [comment]
> @robrap - I don't think docs run on devstack builds, 

What does this mean? That you don’t think you’ll get the banner? Other?

> but does this banner wording work for you?

Thank you. This should imply what people need to do, but if you are open to it, you could add a really exploit note:

> For any fork of devstack, you need to look elsewhere for up-to-date docs.

I’m ok if you think this is overkill, but otherwise, it just makes the seemingly obvious very explicit.

**Comment by sarina** [comment]
I like the additional wording, thanks.

I was looking through the CI tests, and I don't see an automated CI test for docs. I am in a low bandwidth environment and can't do a clone of this repo, so I can't build the docs myself.

File: docs/conf.py [hunk 1]
```diff
@@ -165,6 +165,7 @@
 # documentation.
 #
 html_theme_options = {
+    "announcement": "Devstack is no longer supported by the Open edX community. See <a href='https://docs.tutor.edly.io/index.html'>Tutor</a> for the community-supported developer and installation environment.",
```
Comment (sarina) [inline docs/conf.py]:
```suggestion
    "announcement": "Devstack is no longer supported by the Open edX community. See <a href='https://docs.tutor.edly.io/index.html'>Tutor</a> for the community-supported developer and installation environment.<br><br>For any fork of devstack, you should look elsewhere for up-to-date docs for your fork.",
```

**Review by robrap — APPROVED** [review]
I see. You just mean you can’t test locally? Are you willing to merge and see what happens?

**Comment by sarina** [comment]
I am *so* willing! haha

**Pull request merged**